pkg/web/templates/partials/user/user_create.html

```diff
@@ -1,10 +1,14 @@
-<dialog id="user_pwd_modal" class="modal" open>
+<dialog id="user_modal" class="modal" open>
```

Comment:
The ID has been changed to match the ID of the target element that is replaced when the swap takes place. This should ensure that the ID always appears in the DOM.

Reply:
Good fix!